src/routes/SafeAppLandingPage/components/SafeAppsDetails.tsx

```diff
@@ -19,7 +20,13 @@ type SafeAppDetailsTypes = {
 
 const SafeAppDetails = ({ iconUrl, name, description, availableChains }: SafeAppDetailsTypes): ReactElement => {
   const showAvailableChains = availableChains?.length > 0
-  const { isModalVisible: isLoaded, isSafeAppInDefaultList } = useSecurityFeedbackModal()
+  const { isLoading: isSafeAppListLoading, getSafeApp } = useAppList()
+  const { getAppUrl } = useSafeAppUrl()
+  const url = getAppUrl()
+
+  const isSafeAppInDefaultList = useMemo(() => {
```

Comment:
there's `safeAppDetailsFromConfigService` variable in `src/routes/SafeAppLandingPage/SafeAppLandingPage.tsx` that can be used to determine whether the app is in the default list or not

Reply:
nice!